src/Microsoft.Health.Fhir.Core/Configs/ImportTaskConfiguration.cs

```diff
@@ -101,8 +105,8 @@ public class ImportTaskConfiguration
         public bool DisableOptionalIndexesForImport { get; set; }
 
         /// <summary>
-        /// Disable unique optional index during import data.
+        /// Default not rebuild clustered.
         /// </summary>
-        public bool DisableUniqueOptionalIndexesForImport { get; set; }
+        public bool RebuildClustered { get; set; } = DefaultSqlRebuildClustered;
```

Comment:
>public bool RebuildClustered { get; set; } = DefaultSqlRebuildClustered; [](http://example.com/codeflow?start=8&length=72)

Please remove ability to set rebuild clustered to true

Reply:
Ok, I will remove set function.